examples/md-flexible/src/configuration/MDFlexConfig.h

```diff
@@ -553,6 +561,35 @@ class MDFlexConfig {
    * valueOffset used for cli-output alignment
    */
   static constexpr size_t valueOffset{33};
+
+ private:
+  /**
+   * Stores the particles generated based on the provided configuration file
+   * These particles can be added to the respective autopas container,
+   * but have to be converted to the respective particle type, first.
+   */
+  std::vector<ParticleAttributes> _particles;
```

Comment:
two things here:
1. After the AutoPas object is initialized you can probably flush this vector
2. In the long run it does not seem reasonable to store the particles in the config but rather directly create them in the respective AutoPas objects. If you want to do it as it is implemented now please create an issue and reference it with a TODO here  so that we do not forget about it.

Reply:
I will flush the vector.
I am currently not sure if the way I initialize the particles will work for octrees. So the particle initialization might change soon, anyway